scripts/testimonials.js

```diff
@@ -1,62 +1,36 @@
-const mySiema = new Siema({
-  selector: '.carousel--items',
-  duration: 200,
-  easing: 'ease-out',
-  perPage: 1,
-  startIndex: 0,
-  draggable: true,
-  multipleDrag: true,
-  threshold: 20,
-  loop: false,
-  rtl: false,
-  onInit: () => {},
-  onChange: () => {},
-})
+function setUpCarousel(root) {
+    const siema = new SiemaWithDots({
+        selector: root.querySelector('.carousel--items'),
+        duration: 200,
+        easing: 'ease-out',
+        perPage: 1,
+        startIndex: 0,
+        draggable: true,
+        multipleDrag: true,
+        threshold: 20,
+        loop: true,
+        onInit: function() {
+            this.addDots()
+            this.updateDots()
+        },
+        onChange: function() {
+            this.updateDots()
+        },
+    })
 
-document.querySelector('.carousel--prev').addEventListener('click', () => mySiema.prev());
-document.querySelector('.carousel--next').addEventListener('click', () => mySiema.next());
+    const prevBtn = root.querySelector('.carousel--prev')
+    if (prevBtn) {
```

Comment:
having an if statement here means we can have the option of making carousels without the prev/next buttons and the JS code won't break.